templates/kubernetes/terraform/modules/kubernetes/irsa.tf

```diff
@@ -0,0 +1,48 @@
+# IRSA support: allow application specific policy via service-account and role
+
+# application_policy_list is passed from main.tf as below:
+  # application_policy_list = [
+  #   {
+  #     service_account = "app1"
+  #     namespace       = "piggycloud-me"
```

Comment:
Maybe just "my-app" instead of "piggycloud"? And just one map in the list to make it shorter?

Reply:
done